tests/Behat/Context/FixtureContext.php

```diff
@@ -0,0 +1,74 @@
+<?php
+namespace DNADesign\Elemental\Tests\Behat\Context;
+
+use DNADesign\Elemental\Extensions\ElementalAreasExtension;
+use DNADesign\Elemental\Extensions\ElementalPageExtension;
+use DNADesign\Elemental\Models\BaseElement;
+use DNADesign\Elemental\Models\ElementContent;
+use SilverStripe\BehatExtension\Context\FixtureContext as BaseFixtureContext;
+use SilverStripe\BehatExtension\Utility\StepHelper;
+use SilverStripe\Core\ClassInfo;
+use SilverStripe\Core\Extensible;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\Dev\FixtureBlueprint;
+use SilverStripe\ORM\DB;
+use SilverStripe\ORM\HasManyList;
+
+if (!class_exists(BaseFixtureContext::class)) {
+    return;
+}
+/**
+ * Context used to create fixtures in the SilverStripe ORM.
+ */
+class FixtureContext extends BaseFixtureContext
+{
+    use StepHelper;
+    protected function scaffoldDefaultFixtureFactory()
+    {
+        $factory = parent::scaffoldDefaultFixtureFactory();
+        foreach (ClassInfo::subclassesFor(BaseElement::class) as $class) {
+            $blueprint = Injector::inst()->create(FixtureBlueprint::class, $class);
+            $factory->define($class, $blueprint);
+        }
+        return $factory;
+    }
+    /**
+     * Create a page cothe "Blocks Page" :type has a "Block Title" content element with "Sample content" content
+     *
+     * @Given the :pageTitle :type has a :elementTitle content element with :elementContent content
+     *
+     * @param string $pageTitle
+     * @param string $type
+     * @param string $elementTitle
+     * @param string $elementContent
+     */
+    public function theHasAContentElementWithContent($pageTitle, $type, $elementTitle, $elementContent)
+    {
+        // Create the page (ElementalArea is created on write and attached to it)
+        $targetClass = $this->convertTypeToClass($type);
+
+        $page = $this->getFixtureFactory()->get($targetClass, $pageTitle);
+        if (!$page) {
+            $page = $this->getFixtureFactory()->createObject($targetClass, $pageTitle);
+        }
+        // Ensure that an ElementalArea is created (see ElementalAreasExtension)
+        $page->write();
+
+        // Create the element and assign it to the page's ElementalArea
+        $element = $this->getFixtureFactory()->createObject(ElementContent::class, $elementTitle);
+        $element->ParentID = $page->ElementalAreaID;
+        $element->Content = $elementContent;
+        $element->write();
+
+        $page->ElementalArea()->write();
+    }
```

Comment:
Can you remove this since we aren't using it at the moment? Also the `scaffoldDefaultFixtureFactory` method if it's not needed